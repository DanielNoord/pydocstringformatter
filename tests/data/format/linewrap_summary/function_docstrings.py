def func():
    """A very long summary line that needs to be wrapped. A very long summary line that needs to be wrapped.

    A description that is not too long.
    """


def func():
    """A very long multi-line summary line that needs to be wrapped. A very long multi-line summary line that needs to be wrapped.
    A very long summary line that needs to be wrapped.

    A description that is not too long.
    """


def func():
    """A multi-line summary that can be on one line.
    But it has a new line so it isn't.

    A description that is not too long.
    """


# Since the ending quotes will be appended on the same line this
# exceeds the max length.
def func():
    """A multi-line summary that can be on one line. Something that is just too longgg."""
