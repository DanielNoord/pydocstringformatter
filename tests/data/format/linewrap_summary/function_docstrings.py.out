def func():
    """A very long summary line that needs to be wrapped, A very long summary line that.

    needs to be wrapped.

    A description that is not too long.
    """


def func():
    """A very long multi-line summary line that needs to be wrapped, A very long multi-.

    line summary line that needs to be wrapped.

    A very long summary line that needs to be wrapped.

    A description that is not too long.
    """


def func():
    """A multi-line summary that can be on one line,.

    But it has a new line so it isn't.

    A description that is not too long.
    """


# Since the ending quotes will be appended on the same line this
# exceeds the max length.
def func():
    """A multi-line summary that can be on one line, Something that is just too.

    longgg.
    """


def func():
    """A multi-line summary that can be on one line, Something that is just too.

    long.
    """


def func():
    """A multi-line summary that can be on one line, Something that is just too lon."""


# Regression for bug found in pylint
# We should re-add the quotes to line length if they will never be on the first line.
class LinesChunk:
    """The LinesChunk object computes and stores the hash of some consecutive stripped.

    lines of a lineset.
    """


# Test for multiple periods at the end of the line
def func():
    """A very long summary line that needs to be wrapped, A very long summary line that.

    needs to be wrapp...

    A description that is not too long.
    """
